src/main/io/displayport_oled.c

```diff
@@ -109,7 +109,8 @@ static const displayPortVTable_t oledVTable = {
     .isTransferInProgress = oledIsTransferInProgress,
     .heartbeat = oledHeartbeat,
     .resync = oledResync,
-    .txBytesFree = oledTxBytesFree
+    .txBytesFree = oledTxBytesFree,
+    .supportedTextAttributes = NULL,
```

Comment:
It is, the display driver vtable contains a `supportedTextAttributes` function pointer but the display caches it in a field. We can make the drivers change the field themselves before calling `displayInit()` instead, but it will be a bit less flexible. It saves 16 bytes of flash.

Reply:
Oh, ok, the cached value field name is the same which is confusing. How about refactoring function pointers in the vtable to have a `Fn` suffix?

Reply:
I think that's a bit too much for this PR. What about renaming the field to `cachedSupportedTextAttributes`?

Reply:
Fine for me